src/plugins/intel_cpu/src/compiled_model.cpp

```diff
@@ -23,11 +23,27 @@
 #include <cstring>
 #include <utility>
 
+#if defined(OV_CPU_WITH_ACL)
+#include "nodes/executors/acl/acl_ie_scheduler.hpp"
+#endif
+
 using namespace ov::threading;
 
 namespace ov {
 namespace intel_cpu {
 
+static void initScheduler() {
+#if defined(OV_CPU_WITH_ACL)
+    static std::mutex _mtx;
+    std::lock_guard<std::mutex> _lock{_mtx};
+    static std::once_flag flag_once;
+    std::shared_ptr<arm_compute::IScheduler> acl_scheduler = std::make_unique<ACLScheduler>();
```

Comment:
Why not to simply call make_shared, at least it'll save one dynamic allocation?

Reply:
@maxnick corrected